src/Traveler.js

```diff
@@ -54,13 +55,13 @@ class Traveler {
   }
 
 
-  totalAmountSpent(tripsData, destinationsData) {
-// need to access past and current trips
-// returning a sum so use reduce.
-// need to add up all lodging and flight costs
-//need to add 10% to sum.
+  totalAmountSpent(destinationsData) {
     let neededTrips = [this.pastTrips, this.currentTrip].flat();
-
+    return neededTrips.reduce((total, trip) => {
+      if(trip.date.includes('2020')) {
+        return total += trip.getCostPerTrip(destinationsData);
+      }
+    }, 0);
```

Comment:
I wrote this method in a very different way and I like the route you decided to, nice work.